desktop/src/app-system.css

```
@@ -141,6 +141,18 @@ input,textarea{font-family:inherit;color:inherit}
 .onb-strength-bar.s2::after{width:55%;background:var(--warning)}
 .onb-strength-bar.s3::after{width:80%;background:var(--accent)}
 .onb-strength-bar.s4::after{width:100%;background:var(--success)}
+/* Unrecoverable startup failure: the message is the whole screen, so it gets
+   room to breathe and preserves the newlines the backend formatted it with. */
+.onb-fatal{width:100%;max-height:210px;overflow:auto;white-space:pre-wrap;word-break:break-word;
```

Stylelint (17.14.1)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)